sky/clouds/cloud.py

```diff
@@ -43,6 +43,7 @@ class CloudImplementationFeatures(enum.Enum):
     OPEN_PORTS = 'open_ports'
     STORAGE_MOUNTING = 'storage_mounting'
     HOST_CONTROLLERS = 'host_controllers'  # Can run jobs/serve controllers
+    AUTODOWN = 'autodown'
```

Comment:
How about we just call it `AUTOSTOP` or `AUTO_TEARDOWN` as autostop and autodown should have similar requirements for the cloud to support?

Reply:
Between these two options, I'll change it to `AUTO_TEARDOWN` since it would be confusing for a cloud (Kubernetes) to have `AUTOSTOP` but not `STOP` feature.